vent/types/sql_table.go

```diff
@@ -61,6 +62,11 @@ const (
 	SQLColumnLabelBurrowVer = "_burrowversion"
 	SQLColumnLabelChainID   = "_chainid"
 
+	// errors table
+	SQLColumnLabelErrorsHeight  = "_height"
+	SQLColumnLabelErrorsTxHash  = "_txhash"
+	SQLColumnLabelErrorsMessage = "_result"
```

Comment:
Hm not sure why we need duplicate labels for height and txhash here. Message makes sense but let's call it messge

Reply:
:heavy_check_mark: